### PR TITLE
Data fetch and display finished

### PR DESCRIPTION
**Pull request by Usagib**
Opening this PR for code review and lintfix purposes.

Hello Team! the features implemented were the following:
- Bootstrap UI created
- UX upgraded
- Open weather API data fetched for current weather
- user view info updated
- live deployed
- esLintFixed

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,10 +3,105 @@ import header from './view/header.js';
...
+      }
+      if (weather[0].description === 'snow') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1465220183275-1faa863377e3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1334&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+      if (weather[0].description === 'mist') {
+        document.body.style.background = "#ffffff url('https://images.unsplash.com/photo-1438803235109-d737bc3129ec?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=666&q=80') no-repeat center center fixed";
+        document.body.style.backgroundSize = "cover";
+      }
+    })
```
Comment (Redvanisation) [inline src/index.js]: [OPTIONAL] You could make a Switch statement for that and put it in a separate module, that would be more concise.

File: src/index.js [hunk 2]
```diff
@@ -3,10 +3,105 @@ import header from './view/header.js';
...
+      const showMinMax = document.getElementById('showMinMax');
+      const showWind = document.getElementById('showWind');
+      const showHumidity = document.getElementById('showHumid');
+      const showCity = document.getElementById('showCity');
+      showCity.innerText = 'invalid city, you can search by city and country code (Mexico city, MX)';
+      showTemperature.innerHTML = `check country codes <a href="https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes">Here</a>`;
+      showWeather.innerText = '';
+      showHumidity.innerText = '';
+      showWind.innerText = '';
+      showMinMax.innerText = '';
```
Comment (Redvanisation) [inline src/index.js]: [OPTIONAL] You could just display an error message instead.

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  ♻️ 

### Hello team  🌟

**Nice work on this project, but the following changes need to be done before moving on ⬇️** 

* [x] Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)).

* [x] It includes a rule for the `node_modules` directory on `.gitignore` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/gitignore_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)).

* [x] Files not related to the project should not be committed and added to .gitignore (e.g. `.DS_Store`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).

* [x] Check my comments below.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

File: src/index.js (hunk 1 above)
Comment (Usagib) [reply src/index.js]: Changed if to Switch statement resolved

File: src/index.js (hunk 2 above)
Comment (Usagib) [reply src/index.js]: error messsage shown resolved

**Comment by Usagib** [comment]
Hello Team!. the following changes had been made according to TSE recommendations:
- changed if statement to switch statement on background change
- changed /src/index.js ln:94-105 to display error message instead

As for the changes required i have some comments / questions:
1. Necessary run common commands like build, development should be in package.json inside `scripts'
- package.json already includes run commands as seen in /package.json ln:6-11, is there any configuration i missed?
2. It includes a rule for the node_modules directory on .gitignore
- .gitignore alredy includes reference to node_modules dir as seen in /.gitignore ln:1, is there any other dir reference i must add?

Thank you for the insight. happy coding.

**Comment by MauricioRobayo** [comment]
Great!

Did you make the logo? That's nice!

All previous issues are fixed, you are good to go 🚀 .

Status: __Approved__.

---

[OPTIONAL]: Since you created the [`node_modules`](https://github.com/Usagib/js-weather/tree/03-datafetch/node_modules) directory before including it in `.gitignore` it is on your repo. You might want to remove it, that's why it should be included on `.gitignore` in the first place.

image

**Review by MauricioRobayo — APPROVED** [review]
Great!

Did you make the logo? That's nice!

All previous issues are fixed, you are good to go 🚀 .

Status: __Approved__.

---

[OPTIONAL]: Since you created the [`node_modules`](https://github.com/Usagib/js-weather/tree/03-datafetch/node_modules) directory before including it in `.gitignore` it is on your repo. You might want to remove it, that's why it should be included in `.gitignore` in the first place.

image

**Pull request merged**